design/ceph-csi-driver.md

```diff
@@ -0,0 +1,47 @@
+# Ceph CSI Driver Support
+
```

Comment:
Could you add a comment about what rook release we would be targeting? (similar to "Targeted for v0.9" as seen [here](https://github.com/rook/rook/blob/master/design/decouple-ceph-version.md)). Even though we are dependent on CSI stabilizing, could we target 0.9 to support both CSI and Flex?